bin/lib/zopen-build

```diff
@@ -687,15 +688,51 @@ build()
 check()
 {
   checklog="${LOG_PFX}_check.log"
+  results=2 # Assume the worst
   if [ -n "${ZOPEN_CHECK_CMD}" ]; then
     printHeader "Running Check"
     runAndLog "${ZOPEN_CHECK_CMD} >\"${checklog}\""
-    if ! runAndLog "${ZOPEN_CHECK_RESULTS_CMD}"; then
-      printError "Check failed. Log: ${checklog}"
+    if command -V "${ZOPEN_CHECK_RESULTS}" >/dev/null 2>&1; then
+      ${ZOPEN_CHECK_RESULTS} "${ZOPEN_ROOT}/${dir}" "${LOG_PFX}"
+      results=$?
     fi
   else
     printHeader "Skipping Check"
   fi
+  case "$results" in
+    0) status="Green" ;;
+    1) status="Yellow" ;;
+    2) status="Red" ;;
+    *) status="Red" ;;
+  esac
+  echo "$status" >> "${ZOPEN_INSTALL_DIR}/build.status"
+}
+
+createEnv()
+{
+  cat <<zz >"${ZOPEN_INSTALL_DIR}/.env"
+if ! [ -f ./.env ]; then
+  echo "Need to source from the .env directory" >&2
+  return 0
+fi
+export _BPXK_AUTOCVT=ON
+export _CEE_RUNOPTS="\$_CEE_RUNOPTS FILETAG(AUTOCVT,AUTOTAG) POSIX(ON)"
+export _TAG_REDIR_IN=txt
+export _TAG_REDIR_ERR=txt
+export _TAG_REDIR_OUT=txt
+zz
+
+  if [ -d "${ZOPEN_INSTALL_DIR}/bin" ]; then
+    echo "export PATH=\"\${PWD}/bin:\$PATH\"" >> "${ZOPEN_INSTALL_DIR}/.env"
+  fi
+  if [ -d "${ZOPEN_INSTALL_DIR}/lib" ]; then
+    echo "export LIBPATH=\"\${PWD}/lib:\$LIBPATH\"" >> "${ZOPEN_INSTALL_DIR}/.env"
```

Comment:
should have MANPATH here too

Reply:
added in https://github.com/ZOSOpenTools/utils/pull/52/files#diff-18d8a0f3b6e2204467238e541740a1e6b0092f69dcd8f83bbf3d94627f172e69R733